experiments/standalone/Vegetation/no_vegetation.jl

```diff
@@ -221,11 +221,11 @@ resp = [
     parent(sv.saveval[k].canopy.autotrophic_respiration.Ra)[1] * 1e6 for
     k in 1:length(sol.t)
 ]
-SW_abs = [
+SW_n = [
     parent(sv.saveval[k].canopy.radiative_transfer.SW_n)[1] for
     k in 1:length(sol.t)
 ]
-LW_abs = [
+LW_n = [
```

Comment:
Now, LW_n can be negative because the atmosphere can be colder than the land. 

Then I guess Rn = SW_d - SW_u + LW_d - LW_u.

Reply:
Agreed!